static/app/utils/analytics/performanceAnalyticsEvents.tsx

```diff
@@ -77,6 +78,11 @@ export type PerformanceEventParameters = {
   'performance_views.overview.change_chart': {
     metric: string;
   };
+  'performance_views.overview.has_data': {
+    // either "has_data", "no_data", or "empty" (when the user only sees the onboarding screen)
+    table_data_state: string;
```

Comment:
If there are only 3 possible values, might be worth using a [string literal type union](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types) here. That almost makes the comment superfluous, but the difference between `no_data` and `empty` might not be clear without it.

I don't know if we're already using `empty` elsewhere to mean the onboarding state, but calling it `onboarding` instead might make the meaning more obvious so you don't need to explain it in a comment at all? Just a thought.